privacyscore/backend/management/commands/scanfromfile.py

```diff
@@ -51,10 +41,14 @@ def handle(self, *args, **options):
                 if '.' in url:
                     url = normalize_url(url)
                     site = Site.objects.get_or_create(url=url)[0]
-                    if scan_list:
-                        site.scan_lists.add(scan_list)
                     sites.append(site)
 
+        if options['create_list_name']:
+            self.stdout.write('Creating ScanList {}'.format(options['create_list_name']))
+            scan_list = ScanList.objects.create(name=options['create_list_name'], private=True)
```

Comment:
How about making the private an option for the user, too?  You can let it default to True.

Reply:
In the model, the default value for `private` is False. So default value should be False.

Reply:
Actually, for bulk scans, making them private by default would be a good idea, as this means that they will be hidden in the frontend. We don't want giant lists to show up in the frontend, as this will kill the server if anyone clicks them. So having these private by default would be a good idea. 

Additionally, since we currently can't provide a description for the bulk-scanned list, making them public is not very useful, so I'd argue for keeping them private by default, and if anyone wants to change this, they have access to the admin backend, or else they would not be using the bulk scan interface at all (as it requires SSH access on the main host).

Reply:
I agree with both of the reasons.
Keeping it private by default for now will make things easier until the logic to handle huge lists is in place at-least.

Reply:
what I meant was to introduce another option for calling this command, i.e. `scanfromfile --create-list foo --private /tmp/urls.txt`. But that'd be an extra thing. I'd merge this without that optional flag also.